readme.md

```diff
@@ -27,6 +27,22 @@ cliTruncate('unicorn', 4, {position: 'start'});
 cliTruncate('unicorn', 4, {position: 'middle'});
 //=> 'un…n'
 
+cliTruncate('unicorns rainbow dragons', 20, {position: 'start', preferTruncationOnSpace: true})
```

Comment:
These examples should be added at the `preferTruncationOnSpace` option docs. Already too many examples here.

Reply:
And would be more useful to have one before and after example so users can compare what the options does in different scenarios.